storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3656,8 +3668,17 @@ static int rocksdb_init_func(void *const p) {
       (rocksdb::BlockBasedTableOptions::IndexType)rocksdb_index_type;
 
   if (!rocksdb_tbl_options->no_block_cache) {
-    rocksdb_tbl_options->block_cache =
+    std::shared_ptr<rocksdb::Cache> block_cache =
         rocksdb::NewLRUCache(rocksdb_block_cache_size);
+    if (rocksdb_sim_cache_size > 0) {
```

Comment:
@hermanlee, I am not sure I understand your point

The main purpose of simulated cache is to have a different size than block cache, so that we can know what will be the hit rate if we increase/decrease the size without actually changing it